libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -254,6 +254,7 @@ singleton SRV_Channels method set_output_pwm_chan void uint8_t 0 NUM_SERVO_CHANN
 singleton SRV_Channels method set_output_pwm_chan_timeout void uint8_t 0 NUM_SERVO_CHANNELS-1 uint16_t 0 UINT16_MAX uint16_t 0 UINT16_MAX
 singleton SRV_Channels method set_output_scaled void SRV_Channel::Aux_servo_function_t'enum SRV_Channel::k_none SRV_Channel::k_nr_aux_servo_functions-1 float -FLT_MAX FLT_MAX
 singleton SRV_Channels method get_output_pwm boolean SRV_Channel::Aux_servo_function_t'enum SRV_Channel::k_none SRV_Channel::k_nr_aux_servo_functions-1 uint16_t'Null
+singleton SRV_Channels method get_output_pwm_chan boolean uint8_t'skip_check uint16_t'Null
```

Comment:
So far we have only been skipping checks on floats. Skipping on a uint8 means that we just cast a lua number to uint8. So negative numbers would come out positive and all sorts of other exciting things.

Any reason not to use the PWM by motor function? Its only valid for the first, but motors will only ever output to the first in anycase. Not that this is not a useful binding to add even if you don't need it in this case.